### PR TITLE
COMMERCE-4963 headless-commerce-delivery-cart-impl changed from remot…

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
…e to local service

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartItemResourceImpl.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public CartItem postCartItem(@NotNull Long cartId, CartItem cartItem)
 		ServiceContext serviceContext = _serviceContextHelper.getServiceContext(
 			commerceOrder.getGroupId());
 
-		_cpInstanceService.getCPInstance(cartItem.getSkuId());
```
Comment (marco-leo) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartItemResourceImpl.java:109]: @andrea-ale-sbarra why this call is needed? there is no assignment

Comment (andrea-ale-sbarra) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartItemResourceImpl.java:109]: hey @marco-leo it must be what remains of a previous implementation, I removed it now

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**